app/views/group_orders/_form.html.haml

```diff
@@ -116,6 +116,7 @@
             - quantity_data['used_quantity'] = @ordering_data[:order_articles][order_article.id][:used_quantity]
             - quantity_data['price'] = @ordering_data[:order_articles][order_article.id][:price]
             - quantity_data['minimum_order_quantity'] = @ordering_data[:order_articles][order_article.id][:minimum_order_quantity] unless @ordering_data[:order_articles][order_article.id][:minimum_order_quantity].nil?
+            - quantity_data['maximum_order_quantity'] = @ordering_data[:order_articles][order_article.id][:maximum_order_quantity] unless @ordering_data[:order_articles][order_article.id][:maximum_order_quantity].nil?
```

Comment:
If I'm not mistaken, `@ordering_data[:order_articles][order_article.id][:maximum_order_quantity]` is never set, right? (At least the above fields are all set in [GroupOrder.load_data](https://github.com/foodcoops/foodsoft/blob/d262291ce48447dc404c99ce129c6f3b796efb70/app/models/group_order.rb#L60), but maximum_order_quantity isn't.)